template-settings.php

```diff
@@ -376,7 +375,8 @@
                 </div>
                 
                 
-                <!-- List of past, present, and future vacations scheduled -->
+                <?php // TODO: Add scheduling and history of availability ?>
+                <!-- List of past, present, and future vacations scheduled
```

Comment:
This comment tag is not closed.

Reply:
Yes. I commented out this section for future development. This is part of the conversation this morning, where we are pausing on the scheduling and history part of the availability feature, for possibly after Nov.

Reply:
That makes sense. It just seemed odd to me that the comment tag is not closed.